docs/source/transactions.md

```diff
@@ -0,0 +1,49 @@
+---
+html_meta:
+  "description": "The @transactions endpoint exposes transactions those have been made through the plone website."
```

Comment:
Small language fix: "The @transactions endpoint expose transactions that have been made through the Plone website."